frog/imports/internalLearningItems/li-idea/index.js

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { ReactiveText } from 'frog-utils';
+import Form from 'react-jsonschema-form';
+import { withState } from 'recompose';
+import Button from 'material-ui/Button';
+
+const viewIdea = ({ data }) => (
+  <React.Fragment>
+    <b>{data.title}</b>
+    <br />
+    {data.content.split('\n').map((line, i) => (
+      <React.Fragment key={i}>
+        {line}
+        <br />
+      </React.Fragment>
+    ))}
+  </React.Fragment>
+);
+
+const editIdea = ({ dataFn }) => (
+  <div className="bootstrap">
+    <b>Title:</b>
+    <br />
+    <ReactiveText
+      path="title"
+      dataFn={dataFn}
+      style={{ width: '80%', height: '100%', fontSize: '20px' }}
+    />
+    <br />
+    <br />
+    <b>Content:</b>
+    <br />
+    <ReactiveText
+      path="content"
+      type="textarea"
+      dataFn={dataFn}
+      style={{ width: '80%', height: '100%', fontSize: '20px' }}
+    />
+  </div>
+);
+
+const schema = {
+  type: 'object',
+  properties: {
+    title: {
+      type: 'string',
+      title: 'Idea'
+    },
+    content: {
+      type: 'string',
+      title: 'Text'
+    }
+  }
+};
+
+const creator = withState('formdata', 'setFormdata', {})(
```

Comment:
For `li-idea` is the Create component really different from the Editor component ? You use totally different code.